<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="OpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>467</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleJapaneseFont</family>
     </font>
    </property>
    <property name="text">
     <string>Choose FIle</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>405</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_y">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="prefix">
        <string>Y:  </string>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_z">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="prefix">
        <string>Z:  </string>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_z">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="prefix">
        <string>Z:  </string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_x">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="prefix">
        <string>X:  </string>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_y">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="prefix">
        <string>Y:  </string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_x">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="prefix">
        <string>X:  </string>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Rotate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Filename:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Vertexes count:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>Facets count:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_filename">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_vertexes">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_facets">
       <property name="font">
        <font>
         <family>.AppleJapaneseFont</family>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">openglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
